storage/boltz/indexes.go

```diff
@@ -171,10 +171,10 @@ type SetReadIndex interface {
 }
 
```

Comment:
Make interface methods public so we could add constraint types that are specific to a given store